nrf-hal-common/src/twim.rs

```diff
@@ -86,6 +86,16 @@ where
         Twim(twim)
     }
 
+    /// Enable the instance.
+    pub fn enable(&mut self) {
+        self.0.enable.write(|w| w.enable().enabled());
+    }
+
+    /// Disable the instance.
```

Comment:
This should probably mention what the impact of disabling it is.

Reply:
Addressed in commit 65cbd22.